forc-pkg/src/pkg.rs

```diff
@@ -400,7 +405,7 @@ fn add_deps(
     let fetch_id = fetch_id(&path_map[&graph[*proj_node].id()], fetch_ts);
     let proj_node_after_delete = compilation_order.last().unwrap();
     for added_package in to_add {
```

Comment:
```suggestion
    for (added_dep_name, added_dep_pkg) in to_add {
```
Might be better to avoid the awkward tuple indexing?